File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.PersonService.Models/save-person-unit-library-info_result.ts


import { ISavePersonUnitLibraryInfo_Result } from './isave-person-unit-library-info_result';

export class SavePersonUnitLibraryInfo_Result implements ISavePersonUnitLibraryInfo_Result {
  
	public PersonsUnitLibraryInfoID: number = 0;
	public PersonID: number = 0;
	public UnitID: number = 0;
	public UnitName: string = "";
	public UnitNameEnglish: string = "";
	public JobTitle: string = "";
	public YearsInPosition?: number;
	public WorkEmailAddress: string = "";
	public RankID?: number;
	public RankName: string = "";
	public IsUnitCommander?: boolean;
	public PoliceMilSecID: string = "";
	public HostNationPOCName: string = "";
	public HostNationPOCEmail: string = "";
	public HasLocalGovTrust: boolean = false;
	public LocalGovTrustCertDate?: Date;
	public IsVettingReq: boolean = false;
	public IsLeahyVettingReq: boolean = false;
	public IsArmedForces: boolean = false;
	public IsLawEnforcement: boolean = false;
	public IsSecurityIntelligence: boolean = false;
	public IsValidated: boolean = false;
	public ModifiedByAppUserID: number = 0;
	public ModifiedByAppUser: string = "";
	public UnitParentName: string = "";
	public UnitParentNameEnglish: string = "";
	public AgencyName: string = "";
	public AgencyNameEnglish: string = "";
	public CountryID: number = 0;
  
}

File: GTTS/INL.PersonService.Models/SavePersonUnitLibraryInfo_Result.cs
using System;

namespace INL.PersonService.Models
{
    public class SavePersonUnitLibraryInfo_Result : ISavePersonUnitLibraryInfo_Result
    {
        public long PersonsUnitLibraryInfoID { get; set; }
        public long PersonID { get; set; }
        public long UnitID { get; set; }
        public string UnitName { get; set; }
        public string UnitNameEnglish { get; set; }
        public string JobTitle { get; set; }
        public int? YearsInPosition { get; set; }
        public string WorkEmailAddress { get; set; }
        public int? RankID { get; set; }
        public string RankName { get; set; }
        public bool? IsUnitCommander { get; set; }
        public string PoliceMilSecID { get; set; }
        public string HostNationPOCName { get; set; }
        public string HostNationPOCEmail { get; set; }
        public bool HasLocalGovTrust { get; set; }
        public DateTime? LocalGovTrustCertDate { get; set; }
        public bool IsVettingReq { get; set; }
        public bool IsLeahyVettingReq { get; set; }
        public bool IsArmedForces { get; set; }
        public bool IsLawEnforcement { get; set; }
        public bool IsSecurityIntelligence { get; set; }
        public bool IsValidated { get; set; }
        public int ModifiedByAppUserID { get; set; }
        public string ModifiedByAppUser { get; set; }
        public string UnitParentName { get; set; }
        public string UnitParentNameEnglish { get; set; }
        public string AgencyName { get; set; }
        public string AgencyNameEnglish { get; set; }
        public int CountryID { get; set; }
    }
}
